app/routes/__index/certificate/index.tsx

```diff
@@ -1,11 +1,65 @@
-import { Heading } from '@chakra-ui/react';
+import { Flex, Center, Button, Divider } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+import { useState } from 'react';
+
+import Loading from '~/components/certificate/loading';
+import CertificateDisplay from '~/components/certificate/certificate-display';
+import Request from '~/components/certificate/request';
+import Description from '~/components/certificate/description';
 
 export default function CertificateIndexRoute() {
-  return (
-    <div>
-      <Heading as="h1" size="3xl">
-        Certificate Info or option to request a certificate
-      </Heading>
-    </div>
-  );
+  const [request, setRequest] = useState(false);
```

Comment:
What does `request` represent?  Would this not be `certificate` (i.e., you'd show the certificate display only if you have a certificate)?

Reply:
the 'request' is a state when the certificate has already been requested (true) or not (false)

Reply:
If this is whether or not the certificate has been requested, use that: `certificateRequested`